tests/www/api/experimental/test_dag_runs_endpoint.py

```diff
@@ -128,3 +129,17 @@ def test_get_dag_runs_no_runs(self):
 
         self.assertIsInstance(data, list)
         self.assertEqual(len(data), 0)
+
+
+class TestDagRunsEndpointStoreDAGs(TestDagRunsEndpoint):
+
+    def setUp(self):
+        super().setUp()
+        app, _ = application.create_app(testing=True)
```

Comment:
To generate identical tests, but in different configurations, it is worth using parameterized. I think that only one endpoint should be tested, so it won't be a big problem.

Reply:
Done